.github/workflows/main.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+on: [push]
+jobs:
+  build:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: borales/actions-yarn@v3.0.0
+        with:
+          cmd: install # will run `yarn install` command
+      - uses: borales/actions-yarn@v3.0.0
+        with:
+          cmd: build # will run `yarn build` command
```

Comment:
Some of the subpackage builds are broken (before I made changes to remove the ev.io reference) and it sounds like we aren't going to maintain this fork long term and looking to upstream Fractal wallet into the origin repo. Since we only care about the react-ui component in this repo, can we add these options to install?

`cmd: install --scope @fractalwagmi/wallet-adapter-react-ui`

Same thing for test too and a comment that we aren't maintaining the other packages in here.

Reply:
moved this into the top-level package.json `build` script